pkg/compiler/compiler_test.go

```diff
@@ -51,3 +66,23 @@ func compileFile(src string) error {
 	_, err = compiler.Compile(file)
 	return err
 }
+
+func readPathAndRunFunc(t *testing.T, f func(string, string)) {
+	infos, err := ioutil.ReadDir(examplePath)
+	if err != nil {
+		t.Fatal(err)
```

Comment:
Why not `require`?

Reply:
no idea, it was like that before I didn't touch that. Just refactored.
As I got right you want to have require here. Will do